src/Raven.NewClient/Document/DocumentConvention.cs

```diff
@@ -542,6 +543,7 @@ public JsonSerializer CreateSerializer()
             };
 
             CustomizeJsonSerializer(jsonSerializer);
+            jsonSerializer.Converters.Add(new StringEnumConverter());
```

Comment:
We need this for enum to be string and not int . i will change it not to use  json.net.